RecoLocalMuon/GEMRecHit/src/SealModule.cc

```diff
@@ -16,4 +16,3 @@ DEFINE_FWK_MODULE(ME0RecHitProducer);
 
 #include "RecoLocalMuon/GEMRecHit/src/ME0RecHitStandardAlgo.h"
 DEFINE_EDM_PLUGIN (ME0RecHitAlgoFactory, ME0RecHitStandardAlgo, "ME0RecHitStandardAlgo");
-
```

Comment:
here and in other places it would be better to cleanup spurious empty line changes